### PR TITLE
[MOS-856] Another approach to MTP path in DeviceInfo

### PR DESCRIPTION
**Pull request by SP2FET**
This time it's device-agnostic from purefs paths

**Your checklist for this pull request**

Make sure that this PR:
- [ ] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: module-services/service-desktop/endpoints/deviceInfo/DeviceInfoEndpointCommon.cpp [hunk 1]
```diff
@@ -89,6 +89,15 @@ namespace sdesktop::endpoints
         }
     }
 
+    auto DeviceInfoEndpointCommon::getMtpPath() const -> std::filesystem::path
+    {
+        auto owner = dynamic_cast<ServiceDesktop *>(ownerServicePtr);
```
Comment (Lefucjusz) [inline module-services/service-desktop/endpoints/deviceInfo/DeviceInfoEndpointCommon.cpp:94]: const?

**Review by Lefucjusz — APPROVED** [review]

File: module-services/service-desktop/endpoints/deviceInfo/DeviceInfoEndpointCommon.cpp (hunk 1 above)
Comment (SP2FET) [reply module-services/service-desktop/endpoints/deviceInfo/DeviceInfoEndpointCommon.cpp]: fixed

File: module-services/service-desktop/endpoints/deviceInfo/DeviceInfoEndpointCommon.cpp [hunk 2]
```diff
@@ -83,12 +83,21 @@ namespace sdesktop::endpoints
...
-        auto owner = dynamic_cast<ServiceDesktop *>(ownerServicePtr);
+        const auto owner = dynamic_cast<ServiceDesktop *>(ownerServicePtr);
         if (owner) {
             owner->requestLogsFlush();
         }
     }
 
+    auto DeviceInfoEndpointCommon::getMtpPath() const -> std::filesystem::path
+    {
+        const auto owner = dynamic_cast<ServiceDesktop *>(ownerServicePtr);
```
Comment (marziel) [inline module-services/service-desktop/endpoints/deviceInfo/DeviceInfoEndpointCommon.cpp:94]: [minor] Could be a single `if`

**Review by marziel — APPROVED** [review]

File: module-services/service-desktop/endpoints/deviceInfo/DeviceInfoEndpointCommon.cpp (hunk 2 above)
Comment (SP2FET) [reply module-services/service-desktop/endpoints/deviceInfo/DeviceInfoEndpointCommon.cpp]: fixed

**Pull request merged**